tendrl/commons/objects/cluster/atoms/check_cluster_nodes_up/__init__.py

```diff
@@ -69,6 +68,14 @@ def run(self):
             # no need to check for child job
             return all_node_status_up
         except (etcd.EtcdKeyNotFound, TypeError) as ex:
-            raise FlowExecutionFailedError(
-                "Error checking status of nodes .error: %s" % str(ex)
+            logger.log(
+                "error",
```

Comment:
this is correct do same thing in all place where you are doing return False, log the message and return False

Reply:
yes